assets/js/lib/test-utils/index.jsx

```diff
@@ -49,6 +50,12 @@ export const renderWithRouter = (ui, { route = '/' } = {}) => {
   };
 };
 
+export const withExtendedDefaultState = (component, additionalState = {}) =>
+  withState(component, {
+    ...defaultInitialState,
+    ...additionalState,
+  });
+
```

Comment:
I'd rather `export defaultInitialState` and then compose it using `withState` inside the test

Reply:
Initially I started like that and then it looked natural following the same `with...` pattern, to keep the same level of encapsulation for `defaultInitialState`, composability and to reduce a bit the boiler of merging the default state with a piece of extra state when needed.
However since you prefer that, I changed as per the suggestion so we can proceed.